CHANGELOG.unreleased.md

```diff
@@ -11,6 +11,8 @@
 
 > Users must be able to say: “I had this issue, happy to know it's fixed”
 
+- [Network] Fix created bonded network not linked to the slave hosts [#4529](https://github.com/vatesfr/xen-orchestra/issues/4529) (PR [#4756](https://github.com/vatesfr/xen-orchestra/pull/4756))
```

Comment:
Not sure about the phrasing. Seems a bit unclear to me. What about:
```suggestion
- [Network] Ensure bonds are created on all pool's hosts when creating bonded networks. [#4529](https://github.com/vatesfr/xen-orchestra/issues/4529) (PR [#4756](https://github.com/vatesfr/xen-orchestra/pull/4756))
```
?

Reply:
My PR creates a bond for each pool's hosts instead of ensuring that the bonds are created.